src/nemo_safe_synthesizer/evaluation/statistics/stats.py

```
@@ -145,13 +145,13 @@ def get_numeric_distribution_bins(training: pd.Series, synthetic: pd.Series) ->
     try:
         bins = np.histogram_bin_edges(pd.concat([training, synthetic]), bins="doane", range=(min_value, max_value))
     except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)